phpstan-baseline.neon

```diff
@@ -23121,12 +22646,7 @@ parameters:
 			path: src/Sulu/Bundle/PageBundle/Repository/NodeRepository.php
 
 		-
-			message: "#^Method Sulu\\\\Bundle\\\\PageBundle\\\\Repository\\\\NodeRepository\\:\\:getReferences\\(\\) return type has no value type specified in iterable type PHPCR\\\\NodeInterface\\.$#"
-			count: 1
-			path: src/Sulu/Bundle/PageBundle/Repository/NodeRepository.php
-
-		-
-			message: "#^Method Sulu\\\\Bundle\\\\PageBundle\\\\Repository\\\\NodeRepository\\:\\:getReferences\\(\\) should return array\\<PHPCR\\\\NodeInterface\\> but returns array\\.$#"
+			message: "#^Method Sulu\\\\Bundle\\\\PageBundle\\\\Repository\\\\NodeRepository\\:\\:getReferences\\(\\) should return array\\<PHPCR\\\\NodeInterface\\> but returns array\\<string, PHPCR\\\\PropertyInterface\\>\\.$#"
```

Comment:
this looks a bit worrying, something might be wrong.

getReferences in phpcr returns the list of properties that reference a node.

Reply:
Okay are `References` not `Nodes` are they really `PropertyInterface`: The code is this one: https://github.com/sulu/sulu/blob/1fc46655a4a14afbb2600b6524ff63153d5fd4e1/src/Sulu/Bundle/PageBundle/Repository/NodeRepository.php#L236C35-L236C57

Reply:
the references themselves are properties. they reference a node which you can get with getNode on the property (or a property that you get with getProperty). if you want the referenced uuid/path, you can use getString on the property.

Reply:
phpdoc of NodeInterface::getReferences:

`This method returns all REFERENCE properties that refer to this node,`